src/thinblock.cpp

```diff
@@ -1415,6 +1475,31 @@ void CThinBlockData::DeleteThinBlockBytes(uint64_t bytes, CNode *pfrom)
 
 void CThinBlockData::ResetThinBlockBytes() { nThinBlockBytes.store(0); }
 uint64_t CThinBlockData::GetThinBlockBytes() { return nThinBlockBytes.load(); }
+void CThinBlockData::FillThinBlockQuickStats(ThinBlockQuickStats &stats)
+{
+    LOCK(cs_thinblockstats);
+
+    stats.fEnabled = IsThinBlocksEnabled();
+    if (!stats.fEnabled)
+        return;
```

Comment:
why do we have to track if thinblocks is enabled by setting stats.fEnabled, when we can always get the current value from IsThinBlocksEnabled()?

Reply:
You are correct.  I will remove and directly use the Is*Enabled() calls for thin blocks and graphene in the UI display logic.